pkg/client/testdata/aggregatorpermissions-noncluster-admin.golden

```diff
@@ -185,6 +186,7 @@ metadata:
   labels:
     component: sonobuoy
     sonobuoy-component: aggregator
+    tier: analysis
```

Comment:
Currently confused about these lines; the template had them but it shows I'm adding them now. Want to figure that out before merging.

Reply:
Oh this is on the service not the pod; thats what was added. Removing that.